r2/r2/models/builder.py

```diff
@@ -173,6 +177,21 @@ def wrap_items(self, items):
             except AttributeError:
                 pass
 
+            if hasattr(item, "sr_id") and item.sr_id is not None:
+                w.subreddit = subreddits[item.sr_id]
+
+            if hasattr(item, "link_id") and item.link_id is not None:
+                w.link = links[item.link_id]
+
+            if isinstance(item, Comment):
+                if not hasattr(w, 'subreddit'):
+                    w.subreddit = item.subreddit_slow
+                if not hasattr(w, 'link'):
+                    w.link = item.link_slow
```

Comment:
My reasoning for leaving in these `hasattr` checks are this: While it *should* work normally via lines L180-L184, in that same respect, these checks are done currently via the `add_props` method. Are they actually needed? I don't know. I haven't run into any problems with them not being in here, but just in case I left them.